tests/language/record.liq

```diff
@@ -70,7 +70,11 @@ def f() =
   ignore(list.hd(l).x)
   l = [b,a]
   ignore(list.hd(l).x)
-  def f(c) = [a,c] end      
+  def f(c) = [a,c] end
+
+  # Equality
+  t({a = 5} == {a = 5}, true)
+  t({a = 5} == {a = 6}, false)
```

Comment:
```suggestion
  t({a = 5} == {a = 6}, false)
  t({a = 5, b = 3} == {a = 6}, false)
  t({a = 5} == {a = 6, b = 4}, false)
```

Reply:
Any maybe some more, like positive tests etc.?

Reply:
Argh, I thought that
```
{a = 5, b = 3} == {a = 6}
```
would not type because the two don't have the same type, but I was apparently wrong...

Reply:
That might have consequence on your assumption above?